app/controllers/api/v1/visualization/clusters_controller.rb

```diff
@@ -0,0 +1,168 @@
+          end
+          render json: self.class.get_cluster_viz_data(@study, cluster, params)
+        end
+
+        swagger_path '/studies/{accession}/clusters/{cluster_name}' do
+          operation :get do
+            key :tags, [
+                'Visualization'
+            ]
+            key :summary, 'Get a cluster and its sub-clusters for a study'
+            key :description, 'Get the cluster group and its constituent cluster annotations'
+            key :operationId, 'study_cluster_path'
+            parameter({
+              name: :cluster_name,
+              in: :path,
+              description: 'Name of cluster group',
+              required: true,
+              type: :string
+            })
+            cluster_param_docs.each { |doc| parameter(doc) }
+            cluster_response_docs.each { |doc| response doc[:code], doc }
+          end
+        end
+
+        def show
+          # this endpoint requires a cluster to be specified -- index is used to fetch the default
+          cluster = @study.cluster_groups.find_by(name: params[:cluster_name])
+          if cluster.nil?
+            render json: {error: "No cluster named #{params[:cluster_name]} could be found"}, status: 422 and return
+          end
+          render json: self.class.get_cluster_viz_data(@study, cluster, params)
+        end
+
+        # packages up a bunch of calls to rendering service endpoints for a response object
+        def self.get_cluster_viz_data(study, cluster, url_params)
+          annot_params = {
+            name: url_params[:annotation_name].blank? ? nil : url_params[:annotation_name],
+            type: url_params[:annotation_type].blank? ? nil : url_params[:annotation_type],
+            scope: url_params[:annotation_scope].blank? ? nil : url_params[:annotation_scope]
+          }
+          annotation = ExpressionVizService.get_selected_annotation(study,
+                                                                          cluster,
+                                                                          annot_params[:name],
+                                                                          annot_params[:type],
+                                                                          annot_params[:scope])
+          subsample = url_params[:subsample].blank? ? nil : url_params[:subsample]
+
+
+          coordinates = ClusterVizService.load_cluster_group_data_array_points(study, cluster, annotation, subsample)
+          plot_data = ClusterVizService.transform_coordinates(coordinates, study, cluster, annotation)
+
+          if cluster.is_3d?
+            range = ClusterVizService.set_range(cluster, coordinates.values)
+            if cluster.has_range?
+              aspect = ClusterVizService.compute_aspect_ratios(range)
+            end
+          end
+
+          titles = ClusterVizService.load_axis_labels(cluster)
+
+          axes_full = {
```

Comment:
Consistent formatting of Hashes with rest of controller
```suggestion
          axes_full = {
            titles: titles,
            ranges: range,
            aspects: aspect
          }
```